tests/test_core.py

```diff
@@ -31,9 +32,7 @@ def test_bigchain_class_default_initialization(config):
     from bigchaindb.core import Bigchain
     from bigchaindb.consensus import BaseConsensusRules
     bigchain = Bigchain()
-    assert bigchain.host == config['database']['host']
-    assert bigchain.port == config['database']['port']
-    assert bigchain.dbname == config['database']['name']
+    assert bigchain.connection
```

Comment:
Not really sure what to assert anymore; is just checking that `bigchain.connection` enough?